iati/core/tests/test_resources.py

```diff
@@ -2,14 +2,35 @@
 from lxml import etree
 import pytest
 import six
+import iati.core.constants
 import iati.core.resources
 
 
 class TestResources(object):
     """A container for tests relating to resources."""
 
+    @pytest.fixture(params=[None] + iati.core.constants.STANDARD_VERSIONS)
+    def standard_version_optional(self, request):
+        """Return a list that can be used as variable positional parameters (i.e. `*standard_version_optional`) to test.
+        This can then be used in tests which have an optional parameter for the year.
+
+        Returns:
```

Comment:
Based on the output of `pytest --fixtures iati/`, would suggest that fixtures shouldn't have `Returns:` statements.

Reply:
Hrmm, I see what you mean, but I think that a big advantage of our `Returns:` statements is that they explicitly state what will be returned in a consistent format. Personally, I'd keep them.